olive/passes/onnx/perf_tuning.py

```diff
@@ -33,7 +33,12 @@ def generate_tuning_combos(model, config):
     )
     opt_level_list = config.opt_level_list if config.opt_level_list else [99]
 
-    io_bind_list = [True, False] if config.io_bind else [False]
+    if config.enable_cuda_graph:
```

Comment:
IIRC, you mentioned that cuda_graph need io_bind to accept ortValue. Have you verified that `bind_cpu_input` is work or not?

Reply:
It worked. I have tested